stylesheets_postprocessing.rb

```diff
@@ -4,7 +4,83 @@
 require_relative '../bookmaker/core/header.rb'
 require_relative '../bookmaker/core/metadata.rb'
 
-def evalOneoffs(file, path)
+# ---------------------- VARIABLES
+local_log_hash, @log_hash = Bkmkr::Paths.setLocalLoghash
+
+# Local path vars, css files
+tmp_layout_dir = File.join(Bkmkr::Project.working_dir, "done", Metadata.pisbn, "layout")
+
+pdf_css_file = File.join(tmp_layout_dir, "pdf.css")
+epub_css_file = File.join(tmp_layout_dir, "epub.css")
+
+oneoff_45x7 = File.join(Bkmkr::Paths.scripts_dir, "bookmaker_assets", "pdfmaker", "css", "picador", "oneoff_45x7.css")
+oneoff_45x7_sans = File.join(Bkmkr::Paths.scripts_dir, "bookmaker_assets", "pdfmaker", "css", "picador", "oneoff_45x7_sans.css")
+
+
+# ---------------------- METHODS
+
+def readConfigJson(logkey='')
+  data_hash = Mcmlln::Tools.readjson(Metadata.configfile)
+  return data_hash
+rescue => logstring
+  return {}
+ensure
+  Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+# collect chapterheads
+def getChapters(logkey='')
+	# an array of all occurances of chapters in the manuscript
+	chapterheads = File.read(Bkmkr::Paths.outputtmp_html).scan(/section data-type="chapter"/)
+	return chapterheads
+rescue => logstring
+	return []
+ensure
+  Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+# check for processing instructions for trim
+def getTrimPIs(logkey='')
+	size = File.read(Bkmkr::Paths.outputtmp_html).scan(/<meta name="size"/)
+	unless size.nil? or size.empty? or !size
+		size = File.read(Bkmkr::Paths.outputtmp_html).match(/(<meta name="size" content=")(\d*\.*\d*in \d*\.*\d*in)("\/>)/)[2].gsub(/\s/,"")
+	end
+
+	if size == "4.5in7.125in" and stage_dir == "arc-sans"
```

Comment:
weird, that probably should have errored right?  I'll pass stage_dir to the method and test all the conditions in there to make sure